init

```diff
@@ -1,21 +1,40 @@
 #!/usr/bin/bash
 
+echo "-------Beginning bootstrap script-------"
+
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 HOMEDIR=$(eval echo "~`whoami`")
 
+bash ${SCRIPTDIR}/v2/util/setup-credentials.sh
+
+if [ -f /etc/environment ]; then
+    source /etc/environment
+fi
+
+if [ "$NODE_ROLE" = "control" ]; then
+  echo "-------Control node found, setting up etcd peers-------"
+  sudo bash ${SCRIPTDIR}/v2/util/etcd-peers-dropin.sh
+  sudo systemctl enable "${SCRIPTDIR}/v2/util-units/etcd-peers.service"
```

Comment:
also, I would go one step further and move all of this logic into `v2/util/etcd-peers-dropin.sh`. That's more for aesthetic reasons, but it makes this init file a bit more readable.